.github/workflows/build.yml

```diff
@@ -55,11 +55,4 @@ jobs:
       - name: Clean
         run: bash -x ./build-scripts/cleanup.sh
       - uses: codecov/codecov-action@v3
-      - name: Slack Notification on Failure
```

Comment:
We won't use Slack notification in this moment.

Reply:
```
Secret `SLACK_WEBHOOK` is missing. Please add it to this action for proper execution.
```

Since the error is due to `SLACK_WEBHOOK` not being set, why not skip it if `SLACK_WEBHOOK` is not set?

Reply:
masterを抜けば失敗しなくなってこれを削除する必要もなくなったりします？

どうしてもこれが邪魔なら、`if: failure() && env.SLACK_WEBHOOK != ''`とかしてwebhookのURLが指定されていなかったらスキップのようにしたほうがいいと思います。

それならupstreamでも受け入れてくれるかもしれません。